pypeit/data/arc_lines/lists/KrI_lines.dat

```diff
@@ -3,6 +3,12 @@
 # Creation Date: 2017-Mar-26
 # Creation Date: 2016-Dec-21
 | ion |       wave | NIST | Instr | amplitude |                    Source |
+| KrI |  4275.1719 |    1 |    16 |     10000 |  deimos_1200B_iraf_manual |
+| KrI |  4320.7938 |    1 |    16 |     10000 |  deimos_1200B_iraf_manual |
```

Comment:
Please consider generating all of these files beyond one micron. I had this problem with Argon that you cut off the coverage at 1 micron, but I needed an IR list. It requires someone lots of effort to make these lists from scratch, so I would advocate going up to 2.5um all the time. We usually use OH lines for such calibration but there are situations where this is not the case (GNIRS_XD_10 for example) because the sky lines are too faint.

Reply:
future work